Assets/MixedRealityToolkit.SDK/Features/Utilities/PointerUtils.cs

```diff
@@ -144,6 +144,159 @@ public static IEnumerable<IMixedRealityPointer> GetPointers()
                 }
             }
         }
-    }
 
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the head gaze.
+        /// </summary>
+        /// <param name="ray">The physics ray for the pointer</param>
+        /// <returns>
+        /// True if the ray contains valid data, false otherwise.
+        /// </returns>
+        public static bool TryGetHeadGazeRay(out Ray ray)
+        {
+            return TryGetRay(InputSourceType.Head, Handedness.None, out ray);
+        }
+
+        /// <summary>
+        /// Gets the physics ray based on the interaction enabled pointer assosiciated with the eye gaze.
```

Comment:
Idea: instead of saying physics ray, I would just do a <seealso:Unity.Engine.Physics.Ray/> .

Reply:
fixed